conjur/constants.py

```diff
@@ -8,7 +8,17 @@
 
 # Builtins
 import os
+import platform
+
+# The OS the CLI is run on is determined by the following:
+# See https://stackoverflow.com/questions/1854/python-what-os-am-i-running-on
+# pylint: disable=no-member
+if os.name != "posix" and platform.system() == "Windows":
+    NETRC="_netrc"
+else:
+    NETRC=".netrc"
```

Comment:
Why only NETRC has this logic?
Also, better create a prefix const with this logic, and append the file name to it.
```suggestion
if os.name != "posix" and platform.system() == "Windows":
    INTERNAL_FILE_PREFIX="_"
else:
    INTERNAL_FILE_PREFIX="."
```

And then you will have:
```python
DEFAULT_NETRC_FILE = os.path.expanduser(os.path.join('~', INTERNAL_FILE_PREFIX + "netrc"))
```

Reply:
netrc only has this logic because netrc is a standard not conjur-unique and not conjur generated file like conjurrc

Nice. I like your suggestion

Reply:
> netrc only has this logic because netrc is a standard not conjur-unique and not conjur generated file like conjurrc

I'm not sure I follow you.
from user's perspective, I don't think there's a difference.
is this the correct UX you wish for Windows users?

Reply:
yes windows users should have _netrc. macos/linux should have .netrc
See [here](https://www.labkey.org/Documentation/wiki-page.view?name=netrc) and [here]( https://stackoverflow.com/questions/6031214/git-how-to-use-netrc-file-on-windows-to-save-user-and-password)